### PR TITLE
039: Recreate footer for landing page

### PR DESCRIPTION
**Pull request by averyillson**
First pass for adding the Footer component. 

- Added Background Color
- Font Color
- Footer Component
- Utilized Grid and Column
- Footer Anchor Base && Hover State
- Footer text 'BROUGHT TO YOU BY:'
- 'THE PITTSBURGH COLLECTIVE.' is an external link to (https://thepittsburghcollaborative.com/)
- 'COMMUNITY MOVEMENT BUILDERS.' is an external link to (http://www.communitymovementbuilders.org/)
- 'SWOPE DREAMS.' is an external link to (https://www.swopedreams.com/)

Screenshot 2023-06-13 at 8 04 18 PM

### CODE REVIEW

**Comment by FeederTreeFoundation** [comment]
The footer is present but only on the root page. When navigating to the other links from the root page, the footer disappears. You most likely have to copy the logic in the header to get it to behave the same way.

File: community-stabilization-fund-api/pages/donate/donate.module.css [hunk 1]
```diff
@@ -1,21 +1,24 @@
 .donate {
-  text-align: center;
   margin-top: 12rem;
 }
 
 .donate p {
+  text-align: center;
```
Comment (FeederTreeFoundation) [inline community-stabilization-fund-api/pages/donate/donate.module.css:6]: Good more specific styling!

File: community-stabilization-fund-api/pages/donate/index.tsx [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
+      <a className={styles.btn_link} href='https://communitymovementbuilders.org/donate.html'>
+        <Button className={styles.btn_copy} kind='secondary'>
           CLICK HERE TO DONATE FOR PCSF AT COMMUNITY MOVEMENT BUILDERS
         </Button>
       </a>
-    </div>
-  </BasicLayout>
+    </BasicLayout>
+    <Footer />
+  </div>
```
Comment (FeederTreeFoundation) [inline community-stabilization-fund-api/pages/donate/index.tsx:25]: This is fine for now cause in the refactor we'll put the Footer and Header components all in a Layout component

Comment (averyillson) [reply community-stabilization-fund-api/pages/donate/index.tsx:25]: Understood.

File: community-stabilization-fund-api/pages/donate/donate.module.css (hunk 1 above)
Comment (averyillson) [reply community-stabilization-fund-api/pages/donate/donate.module.css:6]: Understood, we can plan for a more specific rewrite of css going forward.

File: community-stabilization-fund-api/pages/rent-mortgage-utilities-support/index.tsx [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import { Button } from 'carbon-components-react';
 import Link from 'next/link';
 
-import { BasicNavigation } from '../../src/components';
+import { BasicNavigation } from "../../src/components/BasicNavigation";
```
Comment (FeederTreeFoundation) [inline community-stabilization-fund-api/pages/rent-mortgage-utilities-support/index.tsx:4]: Another non-blocking issue but we are exporting packages from the root of folders to be able to import without the explicitness. We should just need to pull components from "@components/" without relative paths and with abstracted paths. I'll be updating the eslint rules soon on it.

File: community-stabilization-fund-api/src/modules/checklists/components/ItemChecklistByBag.tsx [hunk 4]
```diff
@@ -2,13 +2,13 @@ import React from 'react';
...
 import type { FormResponse } from '../../../db';
 
+import { BagLabels } from './BagLabels';
 import { formResponseMock } from '../../../mocks';
 import {
   mapFormResponseToBagItems,
   mapFormResponseToRecipientInfo,
 } from '../utils';
 
-import { BagLabels } from './BagLabels';
```
Comment (FeederTreeFoundation) [inline community-stabilization-fund-api/src/modules/checklists/components/ItemChecklistByBag.tsx:11]: Generally we want to be able to see the components being rendered separately from utils, mocks, hooks, etc for the ease of debugging. I gotta update the eslint rules to respect the difference but putting this here to note it.

**Review by FeederTreeFoundation — APPROVED** [review]

File: community-stabilization-fund-api/src/components/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { Link, Grid, Column } from 'carbon-components-react';
+
+import styles from '../../styles/Home.module.css';
+
+const Footer = () => (
+  <footer className={styles.footer}>
+    <div className={styles.footer_heading}>
+      <Grid>
+        <Column lg={4} md={4} sm={4}><h4>Bought To you by:</h4></Column>
```
Comment (FeederTreeFoundation) [inline community-stabilization-fund-api/src/components/Footer.tsx:9]: Lets make this a p tag instead of a h4

**Pull request merged**